subprojects/antlr/src/integTest/groovy/org/gradle/api/plugins/antlr/AntlrRelocationIntegrationTest.groovy

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.plugins.antlr
+
+import org.gradle.integtests.fixtures.AbstractTaskRelocationIntegrationTest
+
+class AntlrRelocationIntegrationTest extends AbstractTaskRelocationIntegrationTest {
```

Comment:
Does this need to be tested against each of the Antlr versions Gradle supports (2,3,4)?

Reply:
Added tests for each ANTLR versions.